doc/source/api/alleleAnnotations.rst

```diff
@@ -0,0 +1,84 @@
+
+Variant Annotation API
+!!!!!!!!!!!!!!!!!!!!!!
+
+See `Allele Annotation schema <../schemas/alleleAnnotations.html>`_ for a detailed reference.
+
+Introduction
+@@@@@@@@@@@@
+
+Variant alleles can be annotated by comparing them to other reference data sets
+using a variety of algorithms. A standard form of annotation is to compare 
+alleles to a transcript set and calculate the expected functional consequence 
+of the change (e.g. an amino acid change a protein coding transcripts affected 
+by the variant).
+
+This API supports the mining of variant annotations by region or genomic 
+feature and the filtering of the results by predicted functional effect.
+
+Variants Schema Entities
+@@@@@@@@@@@@@@@@@@@@@@@@
+
+The ``VariantAnnotation`` data model, is based on the results provided by variant 
+annotation programs such as VEP, SnpEff and Annovar and others, as well as the 
+VCF's `ANN format <http://snpeff.sourceforge.net/VCFannotationformat_v1.0.pdf>`_ . 
+
+Instead of sending whole set of annotations, the server can send information 
+on specific variants, genomic regions or annotations instead.
+
++---------------------+---------------------------------------------------------------------------------------------------------------------+
+| Record              | Description                                                                                                         |
++=====================+=====================================================================================================================+
+| VariantAnnotationSet| A VariantAnnotationSet record groups VariantAnnotation records. It represents the comparison of a VariantSet to a   |
+|                     | specified reference data set using specified algorithms. It holds information describing the software and reference |
+|                     | data versions use.                                                                                                  |
++---------------------+---------------------------------------------------------------------------------------------------------------------+
+| VariantAnnotation   | A VariantAnnotation record represents the result of comparing a single variant to the set of reference data. It     |
+|                     | contains structured sub-records and a flexible key-value pair ‘info’ field.                                         |
++---------------------+---------------------------------------------------------------------------------------------------------------------+
+| TranscriptEffect    | A transcript effect record describes the effect of an allele on a transcript.                                       |
```

Comment:
'A transcript effect record' =>'A TranscriptEffect' record for consistency - sorry - I know this came from me.